src/commands/validate.ts

```diff
@@ -12,17 +13,27 @@ export default class Validate extends Command {
     help: Flags.help({ char: 'h' }),
     watch: watchFlag(),
     ...validationFlags(),
+    score: Flags.boolean({
+      description: 'Compute the score of the AsyncAPI document',
```

Comment:
Users might might not understand how score is calculated, could we add more description and details about that ?

Reply:
Done @Amzani